tdrs-backend/docs/api/roles.md

```diff
@@ -0,0 +1,146 @@
+
```

Comment:
Documentation for new roles endpoint

Reply:
Might be worth noting that `roles` maps to `Group`s on the backend